### ISSUE TITLE
macro Z_Probe_Accuracy_Hot doesn't work if TEMP param is entered

### ISSUE THREAD

**Issue by jedi1024** [3DPrintDemon/Demon_Klipper_Essentials_Unified#68]
Input:
```
Z_Probe_Accuracy_Hot TEMP=65
```

Error in console:
```
Error evaluating 'gcode_macro Z_Probe_Accuracy_Hot:gcode': TypeError: unsupported operand type(s) for -: 'str' and 'int'
```

It looks like the params.TEMP is not casted to float type before performing subtraction.

**Comment by jedi1024** [comment]
Fix is in PR #69

**Comment by 3DPrintDemon** [comment]
Thanks for the contribution I’ll get this implemented. 😎👍
